javascript/javascript3/week1/weatherApp/weather.js

```diff
@@ -0,0 +1,78 @@
+const body = document.querySelector('body');
+const searchButton = document.getElementById('search-button');
+const input = document.querySelector('input');
+const locationButton = document.getElementById('geo-location');
+
+//Using the input value of the cityname- getting the weather:
+function getWeatherData(){
+let city = input.value;
+    if(!city)
+    {
+    alert(`Please provide a city name`);
+    }
+    else{
+   fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=110d07eec1d93474e56ec4709a2eeb6f`)
+    .then((res) => res.json())
+    .then((data) => {
+        console.log(data);
```

Comment:
please delete any console.log before submitting.

Reply:
Sure- I will do that